src/optimize/babel/register.js

```diff
@@ -1,4 +1,8 @@
 // this file is not transpiled in dev
 
+// this will set babel cache paths as environment variables,
+// it needs to be loaded before babel-register
+const babelOptions = require('./options').node
+
 require('babel-polyfill');
-require('babel-register')(require('./options').node);
+require('babel-register')(babelOptions);
```

Comment:
I think that continuing to depend on the modifications to process.env in `./options` is just going to get us in this spot again at some point. I'm thinking we should move the require calls into `./options`, so `./register` looks something like this:

```js
require('./options').registerNodeOptions();
```